### ISSUE TITLE
The save dialog doesn't announce when it is done

### ISSUE THREAD

**Issue by clapierre** [jupyterlab/jupyterlab#6582]
From Web4All Hackathon https://github.com/diagram-codesprint/jupyterlab/issues/13
The save dialog writes into the status bar that the save has started and also that it has ended. This is not announced by screen reader.

**Comment by tonyfast** [comment]
xref #9399
